scripts/setup-adapters.sh

```diff
@@ -144,7 +144,8 @@ function install_hdfs_deps {
     wget_and_untar https://archive.apache.org/dist/hadoop/common/hadoop-2.10.1/hadoop-2.10.1.tar.gz hadoop
     cp -a hadoop /usr/local/
   fi
-  cmake_install $libhdfs3_dir
+  cd $libhdfs3_dir
```

Comment:
If we don't leave this dir and it is not in () would we not be in the wrong dir at the end of the function?

Reply:
Yes! Good catch!